manual/cops_rails.md

```diff
@@ -41,6 +41,7 @@ it { is_expected.to have_http_status :error }
 
 Name | Default value | Configurable values
 --- | --- | ---
+VersionAdded | `0.54` | Float
```

Comment:
This is wrong. There must be some compartment/namespace bug hiding somewhere.

/cc @Darhazer @bbatsov